src/main/resources/static/js/identifyHandler.js

```diff
@@ -1,5 +1,6 @@
+const hash = new URL(window.location).searchParams.get("hash");
+const tcTokenURLParam = window.location.hash.replace("#", "");
+
 const eidClientButton = document.getElementById("eid-client-button");
-const eidClientBaseUrl = `${
-  eidClientButton.baseURI
-}eID-Client?${window.location.hash.replace("#", "")}`;
-eidClientButton.setAttribute("href", eidClientBaseUrl);
+const eidClientURL = `${eidClientButton.baseURI}eID-Client?${tcTokenURLParam}&hash=${hash}`;
```

Comment:
Why do we add the hash to the eID-Client URL?

Reply:
Because we need it, when the fallback page is loaded and we want to connect that call to the session.

Reply:
💡